test/e2e/e2e.go

```diff
@@ -468,6 +468,11 @@ func createPod(f *framework.Framework, podName, nodeSelector, namespace string,
 	err = e2epod.WaitForPodRunningInNamespace(context.TODO(), f.ClientSet, res)
 
 	if err != nil {
+		res, err = podClient.Get(context.Background(), pod.Name, metav1.GetOptions{})
+		if err != nil {
+			return nil, errors.Wrapf(err, "Failed to get pod %s %s", pod.Name, namespace)
```

Comment:
```suggestion
			return nil, errors.Wrapf(err, "Failed to get pod %s %s", namespace, pod.Name)
```
nit ^ for better readability